package.json

```diff
@@ -6,6 +6,7 @@
     },
     "devDependencies": {
         "browserslist-saucelabs": "^0.1.4",
+        "cms-protractor-helper": ">=0.0.3",
```

Comment:
uhm, why not `~0.0.3` ?
eventually we don't want to break things when api stabilizes in cms-protractor-helper and you're ready to release 1.0